pywhy_graphs/export/tests/test_tetrad.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+import pywhy_graphs
+from pywhy_graphs.export import graph_to_tetrad, tetrad_to_graph
+from pywhy_graphs.testing import assert_mixed_edge_graphs_isomorphic
+
+root = Path(pywhy_graphs.__file__).parents[1]
+filename = root / "pywhy_graphs" / "export" / "tests" / "test_pag_tetrad.txt"
+
+
+def test_to_tetrad(tmp_path):
+    G = tetrad_to_graph(filename, graph_type="pag")
```

Comment:
What happens if a user has `graph_type="dag"` instead here? Will this function detect that there are `o->` edges and prevent that? Clearly the imported file does not represent a DAG so we shouldn't let the user make this mistake. Or we should have clear tests to show what happens when you import a graph under a different graph assumption than the one it was created with

Reply:
I'm going to leave this for another PR.